modeling/model.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modeling
+
+import (
+	"fmt"
+)
+
+// Model wrapper class for the simulation components
+type Model struct {
+	slices    map[string]EndpointSliceGroup
+	alg       RoutingAlgorithm
+	simulator TrafficSimulator
+
+	Region  regionInfo
+	Results []SimulationResult
+	// number of max endpoints per slice
```

Comment:
Nit: Comment should start with field name.

Reply:
fixed